packages/core-components/src/components/MarkdownContent/MarkdownContent.test.tsx

```diff
@@ -62,4 +62,20 @@ describe('<MarkdownContent />', () => {
     expect(fp2).toBeInTheDocument();
     expect(rendered.getByText(');', { selector: 'span' })).toBeInTheDocument();
   });
+  it('Render MarkdownContent component with trusted HTML content', async () => {
+    const rendered = await renderWithEffects(
+      wrapInTestApp(
+        <MarkdownContent
+          content="```<blockquote> May not look like much but got it where it counts</blockquote>```"
```

Comment:
hm isn't this double blockquotes basically? should you remove the triple backticks?